fresh_mm_bot/src/helpers/channel.ts

```
@@ -0,0 +1,80 @@
+                                      : [PermissionsBitField.Flags.ViewChannel],
+                          },
+                      ]
+                    : []),
+            ],
+            ...(parentId ? { parent: parentId } : {}),
+            ...(rateLimitPerUser ? { rateLimitPerUser: rateLimitPerUser } : {}),
+        });
+        if (!channel) throw new Error("Couldn't create channel");
+
+        resolve(channel);
+    });
```

Biome (2.4.10)

[error] Promise executor functions should not be `async`.

(lint/suspicious/noAsyncPromiseExecutor)